website/docs/configuration/managed-configuration.md

```diff
@@ -0,0 +1,380 @@
+---
+sidebar_position: 10
+title: Managed configuration
+description: Using managed configuration to enforce settings in enterprise environments.
+tags: [podman-desktop, configuration, enterprise, managed]
+keywords: [podman desktop, configuration, managed, enterprise, locked, admin, policy]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Managed Configuration
+
+In enterprise environments, system administrators can enforce specific configuration values that users cannot override. This feature allows IT departments to control critical settings such as proxy servers, telemetry policies, and security configurations.
+
+## How it works
+
+The managed configuration system uses three separate JSON files:
+
+1. **User Configuration** - Normal user settings (editable by user)
+2. **Managed Defaults** - Administrator-enforced values (read-only)
+3. **Locked Configuration** - List of keys that must use managed values
+
+When Podman Desktop retrieves a configuration value, it checks these files in priority order:
+
+1. **Locked keys** - Return value from managed defaults (highest priority)
+2. **Unlocked keys** - Return value from user configuration
+3. **Default value** - Return fallback if key doesn't exist
+
+## File locations
+
+<Tabs groupId="operating-systems">
+<TabItem value="linux" label="Linux">
+
+**User Configuration**
+
+- Location: `~/.local/share/containers/podman-desktop/configuration/settings.json`
+- Permissions: User read/write
+- Purpose: Normal user settings configured through the UI
+
+**Managed Defaults**
+
+- Location: `/etc/containers/podman-desktop/default-settings.json`
+- Permissions: Root only
+- Purpose: Administrator-enforced configuration values
+
+**Locked Configuration**
+
+- Location: `/etc/containers/podman-desktop/locked.json`
+- Permissions: Root only
+- Purpose: List of configuration keys that are locked
+
+</TabItem>
+<TabItem value="mac" label="macOS">
+
+**User Configuration**
+
+- Location: `~/Library/Application Support/containers/podman-desktop/configuration/settings.json`
+- Permissions: User read/write
+- Purpose: Normal user settings configured through the UI
+
+**Managed Defaults**
+
+- Location: `/Library/Application Support/com.podman.desktop/default-settings.json`
+- Permissions: Administrator only
+- Purpose: Administrator-enforced configuration values
+
+**Locked Configuration**
+
+- Location: `/Library/Application Support/com.podman.desktop/locked.json`
+- Permissions: Administrator only
+- Purpose: List of configuration keys that are locked
+
+</TabItem>
+<TabItem value="win" label="Windows">
+
+**User Configuration**
+
+- Location: `%APPDATA%\Podman Desktop\configuration\settings.json`
+- Permissions: User read/write
+- Purpose: Normal user settings configured through the UI
+
+**Managed Defaults**
+
+- Location: `C:\ProgramData\com.podman.desktop\default-settings.json`
```

Comment:
Should it be `%PROGRAMDATA%` instead, if the user has different drive they installed windows on

(same comment can be repeated often)